common-controller/internal/config/default_config.go

```diff
@@ -35,5 +35,11 @@ var defaultConfig = &Config{
 		},
 		Environment:       "Default",
 		InternalAPIServer: internalAPIServer{Port: 18003},
+		ControlPlane: controlplane{
+			Enabled:       false,
+			Host:          "localhost",
+			EventPort:     18000,
+			RetryInterval: 5,
+			Persistence:   persistence{Type: "K8s"}},
```

Comment:
what about type ETCD?

Reply:
ideal name is K8s right since ETCD is implementation detail